sinfonier-web/app/views/Drawers/index.html

```diff
@@ -213,6 +213,17 @@
         }, 1001);
       });
     }, 1000);
+    
```

Comment:
It is better to have javascript in separate files, and put condition using data values of html.